server/routes/utils/adHelpers.ts

```diff
@@ -253,3 +254,55 @@ export const getErrorMessage = (err: any) => {
     ? get(err, 'body.error.reason')
     : get(err, 'message');
 };
+
+export const getKibanaErrorResponseByStatusCode = (
+  statusCode: number,
+  message: string
+) => {
+  switch (statusCode) {
+    case 400:
+      return kibanaResponseFactory.badRequest({
```

Comment:
Just curious, why need to explicitly return `kibanaResponseFactory.badRequest`, looks like something duplicate for `400` and `badRequest` Is there any general error response to accept status code ?

Reply:
`kibanaResponseFactory` has multiple response items with corresponding statusCode, `badRequest` has 400 status code. as shown like [this](https://github.com/elastic/kibana/blob/v7.10.0/src/core/server/http/router/response.ts#L155). This method is for given status code, return corresponding response.

But given Tyler's found above, I will revert this change. No need to worry about it.